server/database/get-user.js

```diff
@@ -20,7 +23,7 @@ module.exports = (facebookId, callback) => {
       return callback(null, result.rows[0].user_id);
     }
 
-    dbClient.query(createUser, [ facebookId, ], (insertErr, newUser) => {
+    dbClient.query(createUser, [ newId, facebookId, ], (insertErr, newUser) => {
```

Comment:
What is `newId`? Why is it called `newId`? You are not using it look up users, only create them.. Not really sure what is going on